examples/webgpu_backdrop_water.html

```diff
@@ -228,10 +228,10 @@
 				const scenePassColorBlurred = gaussianBlur( scenePassColor );
 				scenePassColorBlurred.directionNode = waterMask.select( scenePassDepth, scenePass.getLinearDepthNode().mul( 5 ) );
 
-				const vignet = screenUV.distance( .5 ).mul( 1.35 ).clamp().oneMinus();
+				const vignette = screenUV.distance( .5 ).mul( 1.35 ).clamp().oneMinus();
```

Comment:
TBH, I think we should not fix these type of "typos". It's totally normal to shorten variable names and it don't has to be grammatically correct, imo.

I appreciate your efforts here but I suggest you focus on comments for now on. We should not overdo it we these typo fixing...

Reply:
Ok no problem, just trying to help :)

Reply:
@Mugen87 I agree that shortening variable names is OK, but I'd prefer to see spelling errors corrected.

I view the changes here as falling into the later category, and would vote to merge this PR.

Reply:
I'm okay with that. I just want to avoid that we invest too much time in this renaming variables thing. It takes more effort to test/verify such changes than just fixing comments and I want to invest my review time differently, tbh.